sirepo/template/cortex.py

```diff
@@ -19,6 +21,18 @@ def stateful_compute_cortex_db(data, **kwargs):
         return PKDict(
             api_result=sirepo.template.cortex_sql_db.list_materials(),
         )
+    if data.args.api_name == "delete_material":
```

Comment:
Not thrilled with the use of stateful_compute. I think its redundant to say "cortex_db" and maybe call it "sql_db" so we can differentiate from the sirepo db (cortex_db is vague in that way).  Also, duplicating dispatch is problematic. Why not stateful_compute_material_detail, stateful_compute_delete_material, and stateful_compute_list_materials? This would save the dispatch logic and rely on errors handled in template_common.